libs/core/shellbar/shellbar-actions/shellbar-actions.component.ts

```diff
@@ -48,83 +47,75 @@ import { ShellbarUserMenuComponent } from '../user-menu/shellbar-user-menu.compo
  * </fd-shellbar-actions>
  * ```
  */
-
 @Component({
     selector: 'fd-shellbar-actions',
     templateUrl: './shellbar-actions.component.html',
-    encapsulation: ViewEncapsulation.None,
     changeDetection: ChangeDetectionStrategy.OnPush,
     host: {
         '[class.fd-shellbar__group]': 'true',
         '[class.fd-shellbar__group--actions]': 'true'
     },
+    styles: [
+        `
+            :host {
+                flex: 1;
+            }
+        `
+    ],
     standalone: true,
-    imports: [PortalModule, ShellbarActionsMobileComponent, ShellbarActionComponent, ShellbarUserMenuComponent]
+    imports: [PortalModule, ShellbarActionsToolbarComponent, ShellbarActionComponent, ShellbarUserMenuComponent]
 })
 export class ShellbarActionsComponent implements OnDestroy {
     /** The user data. */
-    @Input()
-    user: ShellbarUser;
+    @Input() user: ShellbarUser;
 
     /** The user menu data. */
-    @Input()
-    userMenu: ShellbarUserMenu[];
+    @Input() userMenu: ShellbarUserMenu[];
 
-    /** When set to true, popover list will be closed after selecting the option */
-    @Input()
-    closePopoverOnSelect = false;
+    /** Whether to close the popover after selecting an option. */
+    @Input() closePopoverOnSelect = false;
 
-    /**
-     * Event emitted when search opened.
-     */
-    @Output()
-    searchOpen = new EventEmitter<boolean>();
+    /** Event emitted when the search is toggled. */
+    @Output() searchOpen = new EventEmitter<boolean>();
 
-    /** @hidden */
-    @ContentChildren(FD_SHELLBAR_ACTION_COMPONENT)
-    shellbarActions: QueryList<ShellbarActionComponent>;
+    /** Shellbar action components projected into the content. */
+    @ContentChildren(FD_SHELLBAR_ACTION_COMPONENT) shellbarActions: QueryList<ShellbarActionComponent>;
 
-    /** @hidden */
-    @ContentChild(ShellbarUserMenuComponent)
-    userComponent: ShellbarUserMenuComponent;
+    /** Shellbar user menu component projected into the content. */
+    @ContentChild(ShellbarUserMenuComponent) userComponent: ShellbarUserMenuComponent;
```

Comment:
Please use token instead of the component name

Reply:
refactoring resuest, which should be done under the other task.